app/db.py

```diff
@@ -0,0 +1,81 @@
+import sqlite3
+import random
+from app.product.models import ProductModel
+
+def getNewId():
+    return random.getrandbits(28)
+
+def connect():
+    conn = sqlite3.connect('products.db')
+    cur = conn.cursor()
+    cur.execute("""CREATE TABLE IF NOT EXISTS products 
+                (id INTEGER PRIMARY KEY,
+                product_name TEXT,
+                category TEXT,
+                producer TEXT,
+                description TEXT,
+                price FLOAT)""")
+    
+    # Dodaj kolumnę 'producer', jeśli jeszcze jej nie ma
+    try:
+        cur.execute("ALTER TABLE products ADD COLUMN producer TEXT")
+    except sqlite3.OperationalError as e:
+        # Jeśli kolumna już istnieje, zignoruj błąd
```

Comment:
Jeśli nie uda się dodać do bazy to powinien zostać zwrócony odpowiedni status http

Reply:
git